app/src/main/java/com/example/caregiver/TaskAdapter.java

```diff
@@ -11,22 +11,20 @@
 import android.widget.LinearLayout;
 import android.widget.TextView;
 
-import androidx.recyclerview.widget.ListAdapter;
 
 import java.time.Duration;
-import java.time.Period;
 
-public class TaskAdapter extends ArrayAdapter<Task> {
+public class TaskAdapter extends ArrayAdapter<ViewProgress.Task> {
 
-    public TaskAdapter(Context context, int layout, Task[] array)
+    public TaskAdapter(Context context, int layout, ViewProgress.Task[] array)
     {
         super(context, layout, array);
     }
 
     @Override
     public View getView (int position, View convertView, ViewGroup parent)
     {
-        Task t = getItem(position);
+        ViewProgress.Task t = getItem(position);

```

Comment:
Can you use a more descriptive variable name?

Reply:
This part is not my code. It's @zmcconnell02 . We can do a different PR to review his code when we merge in.